db-ops/secret-application-dev.yml.tpl

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: $USER_NAME
```

Comment:
This template is exactly like the other one, just with a different variable here. How about instead of duplicating it we just change the variable we pass in? (Or remove this and use secrets manager as mentioned above)

Reply:
will use AWS secret manager instead of this one.